src/app/components/CreateLifeForm.tsx

```diff
@@ -0,0 +1,145 @@
+import { Form, Input, DatePicker, Select, Button, notification, Divider } from 'antd';
+import Title from 'antd/lib/typography/Title';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router';
+import { useCreateLifeMutation } from '../api';
+
+const { TextArea } = Input;
+
+const formItemLayout = {
+    labelCol: {
+        xs: { span: 24 },
+        sm: { span: 8 },
+    },
+    wrapperCol: {
+        xs: { span: 24 },
+        sm: { span: 16 },
+    },
+};
+const tailFormItemLayout = {
+    wrapperCol: {
+        xs: {
+            span: 24,
+            offset: 0,
+        },
+        sm: {
+            span: 16,
+            offset: 8,
+        },
+    },
+};
+
+const CreateLifeFrom = () => {
+    const { t } = useTranslation('life');
+    const [form] = Form.useForm();
+    const history = useHistory();
+    const [createLifeMutation, { loading }] = useCreateLifeMutation();
+
+    const onFinish = (values: any) => {
+        const birthday = values.birthday.toISOString();
```

Comment:
No need to manually convert this `toIsoString` this will be automatically done

Reply:
Done